cmd/operator/main.go

```diff
@@ -85,10 +90,30 @@ func Main() int {
 	kubeconfigPath := flagset.String("kubeconfig", "", "The path to the kubeconfig to connect to the apiserver with.")
 	apiserver := flagset.String("apiserver", "", "The address of the apiserver to talk to.")
 	releaseVersion := flagset.String("release-version", "", "Currently targeted release version to be reconciled against.")
+	telemetryConfigFile := flagset.String("telemetry-config", "/etc/cluster-monitoring-operator/telemetry/metrics.yaml", "Path to telemetry-config.")
 	images := images{}
 	flag.Var(&images, "images", "Images to use for containers managed by the cluster-monitoring-operator.")
 	flag.Parse()
 
+	f, err := os.Open(*telemetryConfigFile)
+	if err != nil {
+		fmt.Fprint(os.Stderr, "Could not open telemetry config file.")
+		return 1
+	}
+
+	telemetryConfig := telemetryConfig{}
+	err = yaml.NewYAMLOrJSONDecoder(f, 100).Decode(&telemetryConfig)
+	if err != nil {
+		fmt.Fprint(os.Stderr, "Could not parse telemetry config file.")
+		return 1
+	}
+	f.Close()
+
+	klog.V(4).Info("Configured matches for telemetry:")
+	for _, m := range telemetryConfig.Matches {
+		klog.V(4).Info(m)
```

Comment:
Hmm so we just log the matches nothing else, how come?

Reply:
These are the selectors, so we essentially log the queries at startup that would match what we're sending. Those are typically referred to as matches/matchers.